README.md

```diff
@@ -52,6 +53,10 @@ not have access to the dashboard, please
 - Use `exchange_refresh_token` on your saved `refresh_token` to retrieve a new token
   when your `access_token` expires.
 
+*Note:*
+- CONNECT_ORIGIN (`connect.smartcar.com`): where requests are initially sent to kick-off the Connect/OAuth flow
+- AUTH_ORIGIN (`auth.smartcar.com`): where requests are sent for all token exchanges
+
```

Comment:
Sorry but should this rather go in the [usage section](https://github.com/smartcar/ruby-sdk#usage) as opposed to a random note section.

This is because documentation is for the ENV variable that can be used and not for the constant in code. We can obviously mention the default but it might make more sense to document the ENV var and allow this to be closer to the other ENV var docs. Suggesting something like : 

```bash
# Get your API keys from https://dashboard.smartcar.com/signup
export SMARTCAR_CLIENT_ID=<client id>
export SMARTCAR_CLIENT_SECRET=<client secret>
export SMARTCAR_REDIRECT_URI=<redirect URI>
# Optional ENV variables
# SMARTCAR_CONNECT_ORIGIN (default value : 'connect.smartcar.com') - Used as the host for the URL that starts the Connect/OAuth2 flow
# SMARTCAR_AUTH_ORIGIN (default value : 'auth.smartcar.com') - Used as the host for the token exchange requests.
```

Sorry the reason i am being so particular is because this is a public facing library and the we are a little bit more particular for the public facing SDKs on the documentation front.

I will keep this PR approved

Reply:
No need to apologize. I appreciate the critique and want it to be high quality, no matter how little. Updating now.